### PR TITLE
Use different file for startup in package.json

### PR DESCRIPTION
**Pull request by dhvaneshshah97**

### CODE REVIEW

File: builds/moment-timezone-with-data.min.js [hunk 1]
```diff
@@ -1,7 +1 @@
-//! moment-timezone.js
-//! version : 0.5.13
-//! Copyright (c) JS Foundation and other contributors
-//! license : MIT
-//! github.com/moment/moment-timezone
```
Comment (YizhiHong) [inline builds/moment-timezone-with-data.min.js]: can you keep the header? you can find the latest header in the og version of with-data.min.json

Comment (dhvaneshshah97) [reply builds/moment-timezone-with-data.min.js]: Added!

**Review by jonknack0 — APPROVED** [review]
Business Logic: pass

**Review by YizhiHong — APPROVED** [review]
test: pass

**Pull request merged**